resources/assets/enderio/config/SAGMillRecipes_Core.xml

```diff
@@ -2459,6 +2459,19 @@ addOreDictionaryTooltips=true
     </recipe>
   </recipeGroup>
 
+  <recipeGroup name="Natura">
+
+    <recipe name="Barley" energyCost="800">
+      <input>
+        <itemStack modID="Natura" itemName="barleyFood" />
+      </input>
+      <output>
+        <itemStack modID="EnderIO" itemName="itemPowderIngot" itemMeta="8" number="2" />
```

Comment:
Right now wheat gives 2, and since other mods treat barley and wheat the same, so should EnderIO.